README.md

```diff
@@ -1 +1,312 @@
-# ballerina-awslambda-deployment
\ No newline at end of file
+[![Build Status](https://travis-ci.org/ballerina-guides/ballerina-awslambda-deployment.svg?branch=master)](https://travis-ci.org/ballerina-guides/ballerina-awslambda-deployment)
+
+# Ballerina AWS Lambda Deployment
+[AWS Lambda](https://aws.amazon.com/lambda/) allows you to deploy your code without explicit provisioning or managing servers. This serverless framework works in a way that, you pay only for the time your code is running, so there is no money wasted with idling resources. The AWA Lambda functions can be invoked through events emitting through various services in AWS, such as S3 and SQS. Also, it is possible to invoke it directly using an exposed API through an API gateway.
+
+> In this guide you will learn about how to deploy Ballerina functions as AWS lambda functions.
+
+The following are the sections available in this guide.
+
+- [What you'll build](#what-youll-build)
```

Comment:
```suggestion
- [What you will build](#what-you-will-build)
```
It is a best practice to not use shortened forms in technical writing.

Reply:
Yes, true, I got the basic template from other BBGs in https://ballerina.io/learn/by-guide/. We may have to fix this in all others also.